games/s.yaml

```diff
@@ -1762,6 +1811,20 @@
   updated: 2014-04-16
   url: http://methane.sourceforge.net/
 
+- name: Super Methane Brothers for Wii and GameCube
```

Comment:
Maybe should rebrand this as "for Nintendo homebrew" since I released this for Switch and (at least planned to release) for 3DS.